tests/cases/dom/manipulate/createElement.js

```diff
@@ -0,0 +1,157 @@
+import QUnit from "qunitjs";
+import {createElement} from "../../../../dom/manipulate";
+
+QUnit.module("dom/manipulate/createElement()",
+    {
+        beforeEach: () =>
+        {
+            document.getElementById("qunit-fixture").innerHTML = "";
+        },
+    }
+);
+
+
+QUnit.test(
+    "with one parameter",
+    (assert) =>
+    {
+        const tagName = "div";
+        const className = "identifier";
+        const element = createElement(tagName);
+        element.classList.add(className);
+        document.getElementById("qunit-fixture").appendChild(element);
+
+        const result = document.getElementsByClassName(className);
+        assert.equal(result.length, 1, "element was only created once");
+        assert.equal(result[0], element, `created <${tagName}>`);
+    }
+);
+
+
+QUnit.test(
+    "with one parameter (parameter is custom tag)",
+    (assert) =>
+    {
+        const tagName = "customtagname";
+        const className = "identifier";
+        const element = createElement(tagName);
+        element.classList.add(className);
+        document.getElementById("qunit-fixture").appendChild(element);
+
+        assert.equal(document.getElementsByClassName(className)[0], element, `created <${tagName}>`);
+    }
+);
+
+
+QUnit.test(
+    "with one parameter (first parameter is null)",
+    (assert) =>
+    {
+        assert.throws(
+            () => {
+                const element = createElement(null);
+            },
+            "function threw an error"
+        );
+    }
+);
+
+
+QUnit.test(
+    "with one parameter (first parameter is an empty string)",
+    (assert) =>
+    {
+        assert.throws(
+            () => {
+                const element = createElement("");
+            },
+            "function threw an error"
+        );
+    }
+);
+
+
+QUnit.test(
+    "with two parameters",
+    (assert) =>
+    {
+        const tagName = "div";
+        const className = "identifier";
+        const width = 100;
+        const cssAttributeName = "display";
+        const cssAttributeValue = "none";
```

Comment:
Yeah, you're probably right. It's unnecessarily hard to read the code because of the amount of variables. In addition to that, if I'm going to remove the variables the test is going to contain less lines of code too.